samples/hyperparameter_tuning/README.md

```diff
@@ -4,7 +4,14 @@ This sample demonstrates how to start hyperparameter jobs using your own hyperpa
 
 ## Prerequisites
 
-This sample assumes that you have completed the [common prerequisties](/samples/README.md).
+This sample assumes that you have completed the [common prerequisites](/samples/README.md).
+
+### Upload S3 Data
+
+You will need training data uploaded to an S3 bucket. Make sure you have AWS credentials and are in the proper AWS region for your bucket. Run the following python script replacing YOUR-BUCKET-NAME with the name of your S3 bucket.
```

Comment:
are in the ~~proper AWS region for your bucket~~

and have the bucket in the same region where you plan to create SageMaker resources


---

let's just use
python3 ../training/s3_sample_data.py `$S3_BUCKET_NAME` so the user doesn't need to replace. We assume the user has followed prereqs. If they haven't, the script will anyway throw an error anyway